symfony/plugins/orangehrmAttendancePlugin/Exception/AttendanceServiceException.php

```diff
@@ -46,4 +46,28 @@ public static function punchOutTimeBehindThanPunchInTime(): self
     {
         return new self('Punch Out Time Should Be Later Than Punch In Time');
     }
+
+    /**
+     * @return static
+     */
+    public static function punchInOverlapFound(): self
+    {
+        return new self('Punch-In overlap found');
```

Comment:
```suggestion
        return new self('Punch In Overlap Found');
```
Better to be consistent with other error messages, use title case